CONTRIBUTING.md

```diff
@@ -169,6 +171,10 @@ When declaring a record in XML,
 </record>
 ```
 
+### Records
+
+* For records of model `ir.filters` use explicit `user_id` field.
```

Comment:
The people who will read this doc won't have odoo/odoo#8218 as a reference to understand ;-)